Tasks/VsTest/vstest.ts

```diff
@@ -10,6 +10,7 @@ import * as utils from './helpers';
 import * as outStream from './outputstream';
 import * as ci from './cieventlogger';
 import * as testselectorinvoker from './testselectorinvoker';
+import * as constants from './constants';
```

Comment:
import errorCodes directly.

Reply:
done

---
In reply to: [136024846](https://github.com/Microsoft/vsts-tasks/pull/5156#discussion_r136024846) [](ancestors = 136024846)